src/assorted/validator.js

```diff
@@ -1,12 +1,19 @@
-export default function validator(validatorRules, errorFunction) {
-  for (let i = 0; i < validatorRules.length; i++) {
-    let rule = validatorRules[i];
+/**
+ * Tests a sequence of validation rules, by calling each rule
+ * returning true, if all validations pass.
+ *
+ * @param {Array[function]} validatorRules - List with functions that return a boolean.
+ *                                           They could be for example a ValidateRule.validateFuntion
+ *
+ * @returns {boolean}
+ *
+ */
 
-    if (rule[0]) {
-      errorFunction(rule[1]);
-      return false;
-    }
+export default function validator(validatorRules) {
```

Comment:
It has the name of an object but it's pretty much a predicate function. So maybe `validateRules`? or `rulesAreValid`? or similar?

Reply:
It is, let's go with validateRules, I think that will be more accurate to it's function

Reply:
I like `validateRules` too!